src/components/dapp-staking/dapp/ProjectDetails.vue

```diff
@@ -113,52 +125,55 @@
                 <span class="text--tooltip">{{ $t('common.twitter') }}</span>
               </q-tooltip>
             </button>
+            <button v-if="it.type === CommunityType.Reddit" class="box--share btn--primary">
+              <div class="icon--social">
+                <a :href="it.handle" target="_blank" rel="noopener noreferrer">
+                  <astar-icon-base :icon-name="CommunityType.Reddit">
+                    <astar-icon-reddit />
+                  </astar-icon-base>
+                </a>
+              </div>
+              <q-tooltip>
+                <span class="text--tooltip">{{ $t('common.reddit') }}</span>
+              </q-tooltip>
+            </button>
+            <button class="box--share btn--primary">
+              <div class="icon--social">
+                <a :href="it.handle" target="_blank" rel="noopener noreferrer">
+                  <astar-icon-base viewBox="0 0 18 18" :icon-name="CommunityType.Facebook">
+                    <astar-icon-facebook />
```

Comment:
Hover effect for the Facebook icon looks a bit odd 🤔 CC @Kahonnohak 
![image](https://user-images.githubusercontent.com/92044428/201338058-c4d573f2-5de5-4aaf-8b1c-7503cd693375.png)